content/docs/conditional-rendering.md

```diff
@@ -148,15 +149,15 @@ ReactDOM.render(
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
 
-It works because in JavaScript, `true && expression` always evaluates to `expression`, and `false && expression` always evaluates to `false`.
+जैवसक्रिप्ट मे यह काम करता हे, `true && expression` हमेशा `expression` होता हे, और `false && expression` हमेशा `false` होता हे।
 
-Therefore, if the condition is `true`, the element right after `&&` will appear in the output. If it is `false`, React will ignore and skip it.
+इसीलिए, अगर कोई कंडीशन `true` हे, तोह `&&` के आगे का एलिमेंट आउटपुट में आ जाता हे। अगर यह `false` हे, फिर रियेक्ट इसको इग्नोर कर स्किप करेगा। 
```

Comment:
`हे,` => `है,`

change this everywhere

Reply:
done for two